### ISSUE TITLE
asm test failing

### ISSUE THREAD

**Issue by Ccode-lang** [peregrine-lang/Peregrine#37]
**Describe the bug**
Asm test gives an error.

**To Reproduce**
1. Run test_runner.py

**Expected behavior**
No error.

**Screenshots**
> If applicable, add screenshots to help explain your problem.
```
temp.c: In function ‘main’:
temp.c:279:1: error: impossible constraint in ‘asm’
 __asm__ ( "addl %%ebx, %%eax;" : "=a" ( add ) : "a" ( arg1 ) , "b" ( arg2 ) ) ;
 ^~~~~~~
```

**Desktop (please complete the following information):**
 - OS: [Raspi os]
 - Version [Buster(10)]

**Comment by SaptakBhoumik** [comment]
What is the architecture of your computer?

**Comment by Ccode-lang** [comment]
ohhhhhhhhhh, asm does not work on different computers.

**Comment by SaptakBhoumik** [comment]
> ohhhhhhhhhh, asm does not work on different computers.

Yes. You can ignore it if asm is not working because it is written for intel x86-64

**Comment by SaptakBhoumik** [comment]
I am closing this issue. Open it again if you have other question
